pyproject.toml

```diff
@@ -41,10 +41,10 @@ docs = [
     "sphinxext-opengraph",
 ]
 testing = [
-    "ops-scenario>=7.0.5,<8",
+    "ops-scenario==7.3.0.dev0"
```

Comment:
Heads up @tonyandrewmeyer
This mostly works, we may want to go this way early next cycle.
See the other comment about how I've fixed the o11y/grafana job.